amazon-kinesis-client-multilang/src/main/java/software/amazon/kinesis/coordinator/KinesisClientLibConfiguration.java

```diff
@@ -202,6 +202,11 @@
      */
     public static final int DEFAULT_MAX_LIST_SHARDS_RETRY_ATTEMPTS = 50;
 
+    /**
+     * The number of times the Proxy will retry listShards call when throttled.
+     */
+    public static final long DEFAULT_SCHEDULER_INITIALIZATION_RETRY_INTERVAL_MILLIS = 1000;
```

Comment:
I'm honestly not sure if this file needs changes. It doesn't seem to actually be used by anything but I made the changes nontheless.